packages/react/emotion-utils/src/flex.tsx

```diff
@@ -7,27 +7,31 @@ export interface FlexOptions {
   align?: CSSProperties['alignItems'];
   justify?: CSSProperties['justifyContent'];
   direction?: CSSProperties['flexDirection'];
+  gap?: CSSProperties['gap'];
 }
 
 export function flex(options: FlexOptions): SerializedStyles;
 export function flex(
   align: CSSProperties['alignItems'],
   justify?: CSSProperties['justifyContent'],
-  direction?: CSSProperties['flexDirection']
+  direction?: CSSProperties['flexDirection'],
+  gap?: CSSProperties['gap']
 ): SerializedStyles;
 export function flex(
   alignOrFlexOptions: FlexOptions | CSSProperties['alignItems'],
   justify = 'flex-start',
-  direction = 'row'
+  direction = 'row',
+  gap = 0
 ) {
   if (typeof alignOrFlexOptions === 'object') {
-    const { align = 'stretch', direction = 'row', justify = 'flex-start' } = alignOrFlexOptions;
+    const { align = 'stretch', direction = 'row', justify = 'flex-start', gap = 0 } = alignOrFlexOptions;
 
     return css`
       align-items: ${align};
       display: flex;
       flex-direction: ${direction};
       justify-content: ${justify};
+      gap: ${gap};
```

Comment:
CSSProperties['gap'] can be Property.Gap<string | number> | undefined

It does not have to suffix `px` ?

What if the value already includes px?

Reply:
I missed it. I think I need to think more about this

Reply:
In my Opinion what about this?

```suggestion
      ${gap && `gap: ${gap}`}
```

There is no need to set default value, and user can be modify it by setting typed also
gap value is only setted by user's parameter

Reply:
Sounds like a very good code Thank you, minsu !!

Reply:
I don't understand correctly, but I can check '12px' is apply in my code

I guess you worry about this situation

<img width="220" alt="image" src="https://github.com/toss/slash/assets/57122180/d4cbf48d-67ce-4c55-b417-561fce291fc1">

emotion's css function strip string value.
It is same case about

`justfy: 'center'` is apply `justify-contents: center` not `justify-contents: 'center'`

Reply:
oh very cool !